TesseractOCR/G8Tesseract.mm

```diff
@@ -51,6 +51,22 @@ @implementation G8Tesseract
 
 @synthesize absoluteDataPath=_absoluteDataPath;
```

Comment:
Yup! That was here before, right? One or two commits before. And exactly this line was shown as uncoverred by unit tests. And that's fair, cause that's a setter, which is never called from our code.
That's what I was talking about. Do you know how to cover this code in unit tests? I did propose two ways. @synthesise or self.absoluteDatapath = in the initializer? Which one do you prefer more?